aliases.cwt

```diff
@@ -194,4 +194,12 @@ single_alias[any_plotter_trigger_clause] = {
     count = int
 
     alias_name[trigger] = alias_match_left[trigger]
+}
+
+single_alias[law_ai_chance] = {
+    ### Base chance. 1.0 is 100%.
+    factor = float
+
+    ## replace_scope = { this = title root = title }
+    alias_name[multiplicative_modifier] = alias_match_left[multiplicative_modifier]
```

Comment:
Are this and `decision_ai_will_do_clause` the same? (Other than the scopes, which could be set in the law block)

Reply:
I'm not sure if the ``## replace_scope = { this = title root = title }`` is important or not so I made it a new alias.